index.html

```diff
@@ -0,0 +1,114 @@
+			<div class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+			<div class="availability"> Available</div>
+		</section>
+		<section>
+			<header><h3>Specifications</h3></header>
+			<block class="specs">Size: 4" x 4"</block>
+			<block class="specs">Weight: 8 oz.</block>
+			<footer><h6>Specifications valid until___<h6></footer>
+		</section>
+		<section>
+			<header><h3>Pricing</h3></header>
+			<block class="pricing">1 :1.99</block>
+			<block class="pricing">2-50: $1.49</block>
+			<block class="pricing">51+: $0.99</block>
+		</section>
+	</article>
+	<article class="card">
+		<header><h1>N64</h1></header>
+		<section>
+			<img src="n64.jpg"></img>
+			<div class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+			<div class="availability"> Available</div>
+		</section>
+		<section>
+			<header><h3>Specifications</h3></header>
+			<block class="specs">Size: 4" x 4"</block>
+			<block class="specs">Weight: 8 oz.</block>
+			<footer><h6>Specifications valid until___<h6></footer>
+		</section>
+		<section>
+			<header><h3>Pricing</h3></header>
+			<block class="pricing">1 :1.99</block>
+			<block class="pricing">2-50: $1.49</block>
+			<block class="pricing">51+: $0.99</block>
+		</section>
+	</article>
+	<article class="card">
+		<header><h1>GameCube</h1></header>
+		<section>
+			<img src="gc.jpg"></img>
+			<div class="description">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+			tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+			quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+			consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+			cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+			proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</div>
+			<div class="availability"> Available</div>
+		</section>
+		<section>
+			<header><h3>Specifications</h3></header>
+			<block class="specs">Size: 4" x 4"</block>
+			<block class="specs">Weight: 8 oz.</block>
+			<footer><h6>Specifications valid until___<h6></footer>
```

Comment:
Did you mean to leave the 'Specifications value until____'  blank? From a user standpoint, including it but not specifying is confusing. If it's good indefinitely state so, otherwise include something so the user doesn't think they're missing data ;)